docs/Data/Injector.md

```diff
@@ -0,0 +1,52 @@
+## Module Data.Injector
+
+#### `Injector`
+
+``` purescript
+data Injector a b
+  = Injector (a -> b) (b -> Maybe a)
```

Comment:
I think profunctors is a transitive dependency (via coroutines or freet) so if it is, sounds like a plan!